src/App.tsx

```diff
@@ -1,48 +1,55 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import {
+  NavLink, Navigate, Route, Routes,
+} from 'react-router-dom';
+import { Tabs } from './Components/Tabs';
+import { Tab } from './types/Tab';
 
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+const tabs: Tab[] = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
 
-const App = () => (
-  <>
-    {/* Also requires <html class="has-navbar-fixed-top"> */}
-    <nav className="navbar is-fixed-top has-background-light" data-cy="nav">
-      <div className="navbar-menu">
-        <div className="navbar-start">
-          <a href="/" className="navbar-item isActive">Home</a>
-          <a href="/tabs" className="navbar-item isActive">Tabs</a>
+const App = () => {
+  return (
+    <>
+      <nav className="navbar is-fixed-top has-background-light" data-cy="nav">
+        <div className="navbar-menu">
+          <div className="navbar-start has-navbar-fixed-top">
+            <NavLink to="/" className="navbar-item isActive">Home</NavLink>
+            <NavLink to="/tabs" className="navbar-item isActive">Tabs</NavLink>
```

Comment:
oh, my bad, missed these two – make sure you review the checklist, p. 1

Reply:
added with classNames, but bulma did it also?
 but with 'active' class, 'isActive' was redundunt, I agree)